vault/expiration.go

```diff
@@ -2324,6 +2396,9 @@ func (le *leaseEntry) renewable() (bool, error) {
 	case le == nil:
 		return false, fmt.Errorf("lease not found")
 
+	case le.isZombie():
+		return false, fmt.Errorf("lease is not renewable and has failed previous revocation attempts")
```

Comment:
`lease is expired and has failed previous revocation attempts`

Reply:
good copy pasta catch!